### ISSUE TITLE
Racing king variant incorrect variant_win, result information for a draw

### ISSUE THREAD

**Issue by salvador-dali** [niklasf/python-chess#553]
```
draws = [
  'kr3NK1/1q2R3/8/8/8/5n2/2N5/1rb2B1R w - - 11 14',  # https://lichess.org/9vfn5Vb5
  '1knq1RK1/2n5/8/8/3N4/6N1/6B1/8 w - - 23 25',      #https://lichess.org/hY25qlIi
]

for fen in draws:
  board = chess.variant.RacingKingsBoard(fen)
  print('Is variant end ', board.is_variant_end())
  print('Is variant loss', board.is_variant_loss())
  print('Is variant win ', board.is_variant_win())   # returns True, should be False
  print('Is variant draw', board.is_variant_draw())
  print('Result         ', board.result())  # returns '1-0', should be '1/2-1/2'
  print()
```

`chess.__version__`  '0.31.0'

**Comment by niklasf** [comment]
Thanks for reporting!

**Comment by salvador-dali** [comment]
Thank you Niklasf.
I have not understood, is this considered a valid behavior or will it be fixed?

**Comment by niklasf** [comment]
See above. It's a bug and fixed by d8edcf2f44df92b79f2d2150131f42f86952e6ac.

**Comment by niklasf** [comment]
Fix published in v0.31.1.
